datafusion/optimizer/src/simplify_expressions/guarantees.rs

```diff
@@ -262,18 +260,18 @@ mod tests {
     #[test]
     fn test_inequalities_non_null_bounded() {
         let guarantees = vec![
-            // x ∈ (1, 3] (not null)
+            // x ∈ [1, 3] (not null)
```

Comment:
cc @wjones127 (as I think you added this code originally in https://github.com/apache/arrow-datafusion/pull/7467)